lib/cli/interactive-setup/index.js

```diff
@@ -12,7 +12,11 @@ const steps = {
 };
 
 module.exports = async (context) => {
-  const stepsDetails = new Map();
+  const stepsDetails = new Map(
+    Object.entries(steps).map(([stepName, step]) => {
+      return [stepName, { configuredQuestions: step.configuredQuestions }];
+    })
+  );
```

Comment:
Looks good. I would just keep this object in further processing, and extend it instead of overriding it completely

Currently this doesn't seem to make a difference, but in eventual future changes, overriding this may bite back (we may forget, assign some data in other context and be surprised it's not there)

Reply:
good call